examples/gltf/app.js

```diff
@@ -382,12 +383,15 @@ export class DemoApp {
       this.translate * Math.cos(roll) * Math.cos(-pitch)
     ];
 
-    const uView = new Matrix4()
+    // TODO: find how to avoid using Array.from() to convert TypedArray to regular array
+    const uView = new Matrix4(vrViewMatrix ? Array.from(vrViewMatrix) : null)
```

Comment:
If you are creating a `Matrix4` from an existing array, you are already copying the matrix into the instance. Why would we need another copy before that copy? Did you run into any problems here?

If the matrices were already guaranteed to be provided (as suggested above) you could cut this code.